src/tests/system/tests/test_passkey.py

```diff
@@ -26,6 +26,26 @@ def passkey_requires_root(client: Client) -> tuple[bool, str] | bool:
     return True
 
 
+def umockdev_ipaotpd_update(ipa: IPA) -> bool:
+    """
+    Update the ipa-optd@.service file from ipa server
+    for umockdev
+
+    :return: True or False
+    :rtype: bool
+    """
+
+    ipa_otpd = f"/usr/lib/systemd/system/ipa-otpd@.service"
+    ipa.fs.backup(ipa_otpd)
+    content = "Environment=LD_PRELOAD=/opt/random.so"
+    ipa.fs.append(ipa_otpd, content)
+    for cmd in ["systemctl daemon-reload", "ipactl restart"]:
+        ipa.host.ssh.run(cmd)
+
+    return True
+    # ipa.fs.restore(ipa_otpd)
```

Comment:
I think you can delete this line.   The teardown seems to be properly restoring the ipa-otpd@.service file.